package.json

```diff
@@ -26,12 +27,17 @@
   },
   "homepage": "https://github.com/sparkbox/bouncy-ball#readme",
   "dependencies": {
+    "eslint-config-sparkbox": "^0.1.0",
```

Comment:
I'll move this down to devDependencies, when I make the other dependency changes. 👍